ServiceInnovationLab/FamilyServiceDirectorySearchinator
# Bugfix/security updates

# What has changed and why?
upgraded react-scripts because of security vulnerabilities and then upgrade babel .babelrc to get the existing tests passing

# How to test this change
pull down the branch and run `npm install` and then `npm test` and all tests should pass

- [ ] tested in small screen
